client/components/Windows/SignIn.vue

```diff
@@ -59,6 +59,7 @@
 import storage from "../../js/localStorage";
 import socket from "../../js/socket";
 import RevealPassword from "../RevealPassword.vue";
+import config from "../../../defaults/config";
```

Comment:
If you want to force lowercase usernames, this code needs to be in the server, and use `Helper.config`, not import defaults.